web/packages/plugins/dmt-app/src/pages/SearchPage.tsx

```diff
@@ -421,9 +450,11 @@ function SelectDataSource(props: {
       <MultiSelect
         label="Select data sources to search"
         initialSelectedItems={selectedDataSources}
-        handleSelectedItemsChange={(event: any) =>
-          setDataSources(event.selectedItems)
-        }
+        handleSelectedItemsChange={(event) => {
```

Comment:
```suggestion
        handleSelectedItemsChange={(event) => {
```
Can/should we include a type here?